core/src/main/resources/hudson/model/AbstractProject/configure-common.jelly

```diff
@@ -48,19 +48,17 @@ THE SOFTWARE.
 
   <p:config-assignedLabel />
 
-  <f:section title="${%Advanced Project Options}">
-    <f:advanced>
-      <p:config-quietPeriod />
-      <p:config-retryCount />
-      <p:config-blockWhenUpstreamBuilding />
-      <p:config-blockWhenDownstreamBuilding />
-      <st:include page="configure-advanced.jelly" optional="true" />
-      <f:entry title="${%Display Name}" field="displayNameOrNull">
-         <f:textbox checkUrl="'${rootURL}/checkDisplayName?displayName='+encodeURIComponent(this.value)+'&amp;jobName='+encodeURIComponent('${h.jsStringEscape(it.name)}')"/>
-      </f:entry>
-      <f:optionalBlock name="keepDependencies" checked="${it.keepDependencies}" title="${%Keep the build logs of dependencies}" help="/help/tasks/fingerprint/keepDependencies.html"/>
-    </f:advanced>
-  </f:section>
+  <f:advanced>
+    <p:config-quietPeriod />
+    <p:config-retryCount />
+    <p:config-blockWhenUpstreamBuilding />
+    <p:config-blockWhenDownstreamBuilding />
+    <st:include page="configure-advanced.jelly" optional="true" />
+    <f:entry title="${%Display Name}" field="displayNameOrNull">
+       <f:textbox checkUrl="'${rootURL}/checkDisplayName?displayName='+encodeURIComponent(this.value)+'&amp;jobName='+encodeURIComponent('${h.jsStringEscape(it.name)}')"/>
+    </f:entry>
+    <f:optionalBlock name="keepDependencies" checked="${it.keepDependencies}" title="${%Keep the build logs of dependencies}" help="/help/tasks/fingerprint/keepDependencies.html"/>
+  </f:advanced>
```

Comment:
@jglick cool. Yes, you are correct. This really should have been part of the previous PR about the config page. If you think that means it should come out of this PR, I defer to you. But if you are at least somewhat ambivalent and @recena doesn't object, I want it in somewhere, so now seems as good a time as any, to me.